pkg/cmd/cli/cmd/deploy.go

```diff
@@ -323,6 +323,11 @@ func (o DeployOptions) retry(config *deployapi.DeploymentConfig) error {
 	return nil
 }
 
+// nowFn is extracted from the cancel method for unit testing.
+var nowFn = func() time.Time {
```

Comment:
i wish we have more formal version of this (i think I saw something similar in kube code)... something like an interface